lib/rubocop/cop/style/redundant_return.rb

```diff
@@ -73,14 +73,19 @@ def arguments?(args)
           !args.first.begin_type? || !args.first.children.empty?
         end
 
+        # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
That this causes a `Metrics/CylcomaticComplexity` violation suggests to me that its implementation is flawed.